Sources/Amplitude/Amplitude.m

```diff
@@ -831,16 +855,8 @@ - (void)refreshDynamicConfig {
 }
 
 - (long long)getNextSequenceNumber {
-    NSNumber *sequenceNumberFromDB = [self.dbHelper getLongValue:SEQUENCE_NUMBER];
-    long long sequenceNumber = 0;
-    if (sequenceNumberFromDB != nil) {
-        sequenceNumber = [sequenceNumberFromDB longLongValue];
-    }
-
-    sequenceNumber++;
-    [self.dbHelper insertOrReplaceKeyLongValue:SEQUENCE_NUMBER value:[NSNumber numberWithLongLong:sequenceNumber]];
```

Comment:
If this logic get removed.. do we still have any other places to store the sequenceNumber? Then next time init we would use same sequenceNumber again?

Reply:
Alyssa will handle the NSUserDefaults storing of this data